assets/sass/okta-theme.scss

```diff
@@ -804,7 +827,7 @@
 }
 
 
-.okta-sign-in-tooltip.qtip-custom {
+.okta-tooltip.qtip-custom {
```

Comment:
This classname is added by `qtip` library which is inside `Textbox` component in courage https://github.com/okta/courage/blob/master/src/views/forms/inputs/TextBox.js#L110